### ISSUE TITLE
TypeError global:screen is undefined

### ISSUE THREAD

**Issue by EnzephaloN** [eonpatapon/gnome-shell-extension-caffeine#131]
Using caffein for a while on Debian 9.5 with gnome.
Maybe after an update, caffein stops working. Now I'm on gnome 3.30.2
journalctl says:

    Extension "caffein@patapon.info" had error: TypeError: global.screen is undefined
    Extension "caffein@patapon.info" had error: TypeError: CaffeinIndicator is undefined
    JS Error: Exception in callback for signal " InhibitorRemoved: TypeError: this._objects is undefined
    Caffein<._inhibitorRemovedq/home/xxx/.local/share/gnome-shell/extensions/caffein@patapon.info/extension.js:227:13
    wrapperqresource:///org/gnome/gjs/modules/_legacy.js:82:22
    _emit@resource:///org/gnome/gjs/modules/signals.js:128:27
    _convertToNativeSignal@resource:///org/gnome/gjs/modules/overrides/Gio.js:117:5

**Comment by eggsyntax** [comment]
FWIW, I don't have caffeine installed (just ran across this on a google search for some of those log statements), and I get the same error. I've seen a couple of other pages suggesting that the problem may be in `_legacy.js`. My journalctl:
```
Apr 25 21:04:03 <MY-COMPUTER-NAME> gnome-shell[2942]: JS ERROR: Exception in callback for signal: StatusChanged: Error: value is out of range for Argument 'interval' (type guint32)
                                                    timeout_source@resource:///org/gnome/gjs/modules/mainloop.js:65:13
                                                    timeout_add@resource:///org/gnome/gjs/modules/mainloop.js:81:12
                                                    _onStatusChanged@resource:///org/gnome/shell/ui/screenShield.js:851:35
                                                    wrapper@resource:///org/gnome/gjs/modules/_legacy.js:82:22
                                                    _init/<@resource:///org/gnome/shell/ui/screenShield.js:511:13
                                                    _emit@resource:///org/gnome/gjs/modules/signals.js:128:27
                                                    _convertToNativeSignal@resource:///org/gnome/gjs/modules/overrides/Gio.js:117:5
```

**Comment by eggsyntax** [comment]
(not looking for help on it here, of course! Just thought it might be useful to y'all to know that it doesn't seem to be a problem specific to this extension)

**Comment by jonafato** [comment]
It looks like this was fixed in #130. Can this issue be closed?
